beta-src/src/state/game/game-api-slice.ts

```diff
@@ -143,14 +144,25 @@ export const markMessagesSeen = createAsyncThunk(
     gameID: string;
     seenCountryID: string;
   }) => {
-    const { data } = await getGameApiRequest(
+    const { data } = await postGameApiRequest(
       ApiRoute.MESSAGES_SEEN,
       queryParams,
     );
     return data as string;
   },
 );
 
+export const setBackFromLeft = createAsyncThunk(
```

Comment:
I personally would not name these `set` as they can be confused with state setters. For consistency with the other function you could call it `markBackFromLeft`.

I like `update` for a prefix for server updates, and `fetch` for server gets.

In this case I would name the function `markReturnedToGame`

Also, probably it should be a `put` not a `post`, because (unless I'm missing something) this particular request is idempotent.